base/string_utils.cpp

```diff
@@ -447,4 +447,75 @@ void ParseCSVRow(std::string const & s, char const delimiter, std::vector<std::s
     target.clear();
 }
 
+/*
+ * Extract username from socialPage.
```

Comment:
base/string_utils is the wrong place for this code. The right place should be somewhere in the editor validators.